### PR TITLE
[14.0][mig] sale_invoice_policy

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: sale_invoice_policy/wizards/res_config.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _name = "res.config.settings"
+    _inherit = "res.config.settings"
+
+    sale_default_invoice_policy = fields.Selection(
+        related="default_invoice_policy",
+        default_model="product.template",
```
Comment (i-vyshnevska) [inline sale_invoice_policy/wizards/res_config.py]: is "default_model" have an effect when the field is related?

**Comment by damdam-s** [comment]
override #1385 and  #1679 

refactor due to cache issue w/ v14. leading to module not being installable.

configuration moved from res.config.settings to ir.config_parameter

**Comment by HugOdoo** [comment]
Hello, 

Where are we on the merging of this?

Let me know,

Thanks,
Hugo

**Comment by damdam-s** [comment]
needs review/testing

**Comment by enriquemartin** [comment]
@damdam-s @ArcheTI-HH Have a look, I have added some improvements in addition to the migration itself https://github.com/OCA/sale-workflow/pull/1787

**Comment by rafaelbn** [comment]
This PR is duplicated with https://github.com/OCA/sale-workflow/pull/1385 which was done before. If @sebastienbeau doesn't answer we can maintain the more active 😄  Thank you !

**Comment by enriquemartin** [comment]
Hi @rafaelbn you can check my PR [#1787](https://github.com/OCA/sale-workflow/pull/1787), which has some reviews and is ready to be merged.

**Comment by rafaelbn** [comment]
OMG! We have 3 PR with the same module!

- https://github.com/OCA/sale-workflow/pull/1385
- https://github.com/OCA/sale-workflow/pull/1762
- https://github.com/OCA/sale-workflow/pull/1787

🤔 Dear great contributors 😄  @damdam-s @enriquemartin make more that 1 PR per module is not optimal, before making a PR you should please look for other already done. The first PR has preference except it is abandoned. If there is a new PR with the same module and you continue in the new one, please close the old one 🙏 . In other case it's really complicated to maintain.

Please close the PR which "you" opened and you are not going to work any more

Thank you!

**Comment by rousseldenis** [comment]
@damdam-s You reviewed the new one, we can close this ?

**Comment by damdam-s** [comment]
yes of course

**Comment by rafaelbn** [comment]
@rousseldenis do you know which one is the good one?

https://github.com/OCA/sale-workflow/pull/1787 or https://github.com/OCA/sale-workflow/pull/1385

**Pull request closed without merging**